src/ripple/consensus/ConsensusParms.h

```diff
@@ -70,8 +70,16 @@ struct ConsensusParms
     // Consensus durations are relative to the internal Consensus clock and use
     // millisecond resolution.
 
-    //! The percentage threshold above which we can declare consensus.
+    //! The percentage threshold and floating point factor above which we can
+    //! declare consensus.
     std::size_t minCONSENSUS_PCT = 80;
+    float minCONSENSUS_FACTOR = static_cast<float>(minCONSENSUS_PCT / 100.0f);
```

Comment:
This logic has always been a floating point. The change I made calculates it into a variable so that it can be re-used in the code. Anyway, I've never heard of std::ratio until right now and prefer not to re-implement perfectly functioning consensus code as my first use of it.